dags/common/utils.py

```diff
@@ -198,3 +200,5 @@ def check_dagrun_state(dagrun: DagRun, not_allowed_states=[], allowed_states=[])
         value = dagrun.get_state() == dag_run_states[allowed_state]
         states_values.append(value)
     return all(states_values)
+def generate_ids():
+    return "".join(random.choices(string.ascii_uppercase + string.digits, k=10))
```

Comment:
I think it's better to use uuid module https://docs.python.org/2/library/uuid.html#module-uuid

Reply:
Is it better to install the modules, if I am using it just in one place?

Reply:
uuid is built in in python